scripts/overlap/kubectl-kubestellar-get-config-for-space

```diff
@@ -78,19 +78,18 @@ if [ -z "$sm_core_config" ]; then
     exit 1
 fi
 
+if [ -z "$space_config_file_name" ]; then
```

Comment:
Since this is now a required parameter, I would make it a positional parameter rather than a flag.

Reply:
Isn't it more readable with the flags explicitly specifying what each input represents?

Reply:
If you feel this is a big mistake.  Please ping me to discuss.